### PR TITLE
Updated additional files in sample viewer source code tab

### PR DESCRIPTION
**Pull request by duffh**
# Description

Fixed incorrectly showing or missing code in the source code tab in the sample viewers.

## Type of change

- Bug fix
- Sample viewer enhancement

## Platforms tested on

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI WinUI
- [x] MAUI Android
- [x] MAUI iOS
- [x] MAUI MacCatalyst

## Checklist

- [x] Self-review of changes
- [x] All changes work as expected on all affected platforms
- [x] There are no warnings related to changes
- [x] Code is commented and follows .NET conventions and standards
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] No unrelated changes have been made to any other code or project files
- [ ] Screenshots are correct size and display in description tab (800 x 600 on Windows, MAUI mobile platforms use the MAUI Windows screenshot)

### CODE REVIEW

**Review by williambohrmann3 — APPROVED** [review]
Got it, we want users to see codebehind by default not xaml (I agree). Did a user request this, is this backed by analytics, design council choice, or just logical decision?

**Comment by williambohrmann3** [comment]
So snippets in `metadata.json` files aren't responsible for the display order (I misunderstood). What was the benefit of flipping the order of `.xaml` and `.cs` files in the snippets section?

File: src/WPF/WPF.Viewer/Samples/Layers/WMSLayerUrl/readme.metadata.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
         "WmsLayer"
     ],
     "snippets": [
-        "WMSLayerUrl.xaml.cs",
-        "WmsLayerUrl.xaml"
+        "WmsLayerUrl.xaml",
```
[the remark this answers is not in the document]
Comment (duffh) [reply src/WPF/WPF.Viewer/Samples/Layers/WMSLayerUrl/readme.metadata.json]: This should be rectified with the recently pushed changes.

**Comment by duffh** [comment]
> So snippets in `metadata.json` files aren't responsible for the display order (I misunderstood). What was the benefit of flipping the order of `.xaml` and `.cs` files in the snippets section?

Snippets are responsible for the display order on the developer website (from what I have seen). We want to put an emphasis on our code when it leverages the Maps SDK API rather than UI code, converters and helpers. Organizing our snippets in this way should provide a better user experience.

**Review by amweis226 — APPROVED** [review]
Confirmed working on MAUI iOS (iPad and iPhone)

**Pull request merged**